scipy/fft/_realtransforms.py

```diff
@@ -1,15 +1,34 @@
-from ._basic import _dispatch
-from scipy._lib.uarray import Dispatchable
+from scipy._lib._array_api import array_namespace
+from . import _realtransforms_uarray
 import numpy as np
 
 __all__ = ['dct', 'idct', 'dst', 'idst', 'dctn', 'idctn', 'dstn', 'idstn']
 
 
-@_dispatch
+def _execute_1D(uarray_func, x, type, n, axis, norm,
+                overwrite_x, workers, orthogonalize):
+    xp = array_namespace(x)
+    x = np.asarray(x)
+    y = uarray_func(x, type=type, n=n, axis=axis, norm=norm,
+                    overwrite_x=overwrite_x, workers=workers,
+                    orthogonalize=orthogonalize)
+    return xp.asarray(y)
+
+
+def _execute_nD(uarray_func, x, type, s, axes, norm, 
+                overwrite_x, workers, orthogonalize):
+    xp = array_namespace(x)
+    x = np.asarray(x)
+    y = uarray_func(x, type=type, s=s, axes=axes, norm=norm,
+                    overwrite_x=overwrite_x, workers=workers,
+                    orthogonalize=orthogonalize)
+    return xp.asarray(y)
+
+
 def dctn(x, type=2, s=None, axes=None, norm=None, overwrite_x=False,
          workers=None, *, orthogonalize=None):
-    """
-    Return multidimensional Discrete Cosine Transform along the specified axes.
+    """Return multidimensional Discrete Cosine Transform
+    along the specified axes.
```

Comment:
can you undo this change? it's normally a single line, so the original is preferred

Reply:
Ah sorry, I misread the PEP. I read

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.

but failed to read

> The summary line may be on the same line as the opening quotes or on the next line.

Reply:
Okay, I'll make a PR to update the dev docs. https://docs.scipy.org/doc/scipy/dev/contributor/pep8.html#pep8-scipy says

> It is recommended to leave existing style issues alone unless they exist in files you are already modifying.

But perhaps "unless they exist in _lines of code_ you are already modifying" better communicates what you are describing.